libmamba/src/api/clean.cpp

```diff
@@ -292,6 +293,14 @@ namespace mamba
             }
         }
 
+        if (clean_force_pkgs_dirs)
+        {
+            for (auto& cache: ctx.pkgs_dirs)
```

Comment:
maybe better to use `            for (auto* pkg_cache : caches.writable_caches())`?